modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/UpgradeLayoutFeatures.kt

```diff
@@ -136,7 +152,25 @@ internal fun UpgradeLayoutFeatures(
             }
 
             if (canUpgrade) {
-                SubscribeButton(onClickSubscribe)
+                Column(
+                    modifier = Modifier,
+                    verticalArrangement = Arrangement.Center,
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                ) {
+                    val bottomPadding = if (shouldShowOffer) 0.dp else 50.dp
```

Comment:
Do we need this much padding if we aren't showing the offer? This screenshot I found of the old design didn't but much?

<img width="538" alt="Screenshot 2024-08-21 at 5 08 26 PM" src="https://github.com/user-attachments/assets/2c48d488-61a9-463d-b46e-3b2d41dbe6b7">

Reply:
I copied the same bottom padding from here PviUP0aiZctOprDuuYRwpb-fi-2919_51645 and update it